tests/foreman/ui/test_remoteexecution.py

```diff
@@ -404,6 +428,12 @@ def test_positive_run_default_job_template(self):
                     job_template='Run Command - SSH Default',
                     options_list=[{'name': 'command', 'value': 'ls'}]
                 )
+                # get job invocation id from the current url
```

Comment:
I believe @pondrejk left it this way because we see the output in the robottelo.log. However I agree that this should be printed out as a part of `AssertError` message.